rcmd/testsite/golden/simple/pkgr.yml

```diff
@@ -0,0 +1,12 @@
+Version: 1
```

Comment:
This whole golden directory can be removed, they were from when I was trying to update the older tests.